### ISSUE TITLE
no categories

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#774]
```
What's the problem?
It doesn't recognize categories anymore. Besides, some it get stuck when you 
try to publish a post. and if I save the progress, it doesn't keep it as a 
draft.

What browser are you using?
Chrome

What version of ScribeFire are you running?
4.2
```

-----
Original issue reported on code.google.com by `enzo.ang...@gmail.com` on 3 Sep 2013 at 9:55

**Comment by GoogleCodeExporter** [comment]
```
What blogging platform are you using?
```

-----
Original comment by `scribefi...@gmail.com` on 4 Sep 2013 at 8:21

**Comment by GoogleCodeExporter** [comment]
```
Wordpress
```

-----
Original comment by `enzo.ang...@gmail.com` on 4 Sep 2013 at 8:24

**Comment by GoogleCodeExporter** [comment]
```
This is fixed  posting in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:33
* Changed state: **Fixed**
